backend/package.json

```diff
@@ -25,8 +25,8 @@
   },
   "dependencies": {
     "@cere-ddc-sdk/smart-contract": "0.5.0",
-    "@polkadot/api": "4.2.1",
-    "@polkadot/api-contract": "4.2.1",
+    "@polkadot/api": "7.15.2-15",
```

Comment:
Check with @MRamanenkau needed version.

Reply:
@AndreiNavoichyk, @MRamanenkau  why can't we use the latest `@polkadot/api` version `9.8.2`?

Reply:
fixed